issues/issue_OCPQE_5204.yaml

```diff
@@ -0,0 +1,4 @@
+description: 'https://issues.redhat.com/browse/OCPQE-5204'
```

Comment:
Is the intention here to get known issues no matter which test case id the known issue is logged for as long as the string is matched? 
I don't see an association between the known issue and testcase id the known issue is logged for which is fine as long as that is the intention.

Reply:
Correct there is no requirement that an issue be mapped 1:1 to a test case. Many bugs or infrastructure issues can cause failures across multiple test cases.